grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_query_helper/osquery_query_helper.scss

```diff
@@ -0,0 +1,17 @@
+// Styles declared here are effectively global because of ViewEncapsulation.None
```

Comment:
Why do we need to have these styles defined as global? They all seem to be used inside osquery_query_helper.ng.html and not inside any of its child components.

Reply:
Because of `mat-autocomplete`. I will update the comment so that it describes the reason. This issue is relevant: https://github.com/angular/components/issues/11764#issuecomment-400370834